### PR TITLE
Add logs to cache hooks events

### PR DESCRIPTION
**Pull request by hectorgomezv**
Changes:
- Adds a log message for each event received by `CacheHooksService` in order to improve traceability.

### CODE REVIEW

File: src/routes/cache-hooks/cache-hooks.service.ts [hunk 1]
```diff
@@ -275,14 +287,62 @@ export class CacheHooksService {
...
+        this._logEvent(event);
         break;
     }
     return Promise.all(promises);
   }
+
+  private _logSafeTxEvent(
+    event: ExecutedTransaction | NewConfirmation | PendingTransaction,
+  ): void {
+    this.loggingService.info({
```
Comment (fmrsabino) [inline src/routes/cache-hooks/cache-hooks.service.ts:307]: I would move this to `debug` level given the scope of this change.

Comment (fmrsabino) [inline src/routes/cache-hooks/cache-hooks.service.ts:307]: I would add a "type" for this log for grouping and filtering these logs.

**Review by fmrsabino — APPROVED** [review]

File: src/routes/cache-hooks/cache-hooks.service.ts (hunk 1 above)
Comment (hectorgomezv) [reply src/routes/cache-hooks/cache-hooks.service.ts:309]: Added in [d628fae](https://github.com/safe-global/safe-client-gateway/pull/852/commits/d628fae78be188e82ba36dc8ad022fabe899e41b)

**Review by fmrsabino — APPROVED** [review]

File: src/routes/cache-hooks/cache-hooks.service.ts (hunk 1 above)
Comment (hectorgomezv) [reply src/routes/cache-hooks/cache-hooks.service.ts:309]: Adding these lines to `debug` level, and enabling that level, would generate a lot of logging, and I think this change could be permanent so I'm more leaning towards using `info` if you agree 🙂

**Pull request merged**